src/routes/teams.js

```diff
@@ -1018,3 +1041,26 @@ function convertKeys(input, method) {
     }
     return output;
 }
+
+async function getMaxTeamInvites({ user, team, server }) {
```

Comment:
```suggestion
async function getMaxTeamInvites({ user, server }) {
```

`team` is not used in this function.

Reply:
correct

Reply:
it's actually the other way around. it's `user` which isn't needed. `team` has to be passed to the event, because the `subscriptions` plugin needs to know the team in order to decide if a limit applies or not

Reply:
Oh true! And you actually only need the team ID so you could only pass `params.id` there.